src/commands/bundle.ts

```diff
@@ -0,0 +1,81 @@
+import { Flags } from '@oclif/core';
+import { Example } from '@oclif/core/lib/interfaces';
+import Command from '../base';
+import bundle from '@asyncapi/bundler';
+import fs from 'fs';
+import path from 'path';
+import { ErrorLoadingSpec } from '../errors/specification-file';
+
+export default class Bundle extends Command {
+  static description = 'Bundle multiple asyncapi files together.';
```

Comment:
is it only for multiple files, what if I have one file, it still bundles references right?

Reply:
Yeah, it first picks up a single file resolve references (reference it into components if required) then combines it with other spec files (which have gone through the same process).